### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mcrd25**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <div id="main-wrapper">
+        <header>
+            <div id="signin-link">
+                <div id="forgot-pwd">
+                    <span>Already have an Intuit account?</span>
+                    <span><a href="#">I forgot my User ID or Password</a></span>
+                </div>
+                <div id="signin-btn">
+                    <a href="#">
+                        <button id="signin">
```
Comment (heyapricot) [inline index.html]: The element button must not appear as a descendant of the `<a>` element.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                        Sign In
+                        </button>
+                    </a>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div id="header">
+                <div id="brand">
+                    <img id="brand-name" src="" alt=""> INTUIT
```
Comment (heyapricot) [inline index.html]: Bad value for attribute src on element img: Must be non-empty.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                        <img id="qb-txt" class="svg-brand-txt" src="https://accounts.intuit.com/IUS-Plugins/v2/images/quick-books-text.svg" alt="">
+                    </a>
+                    <a href="" id="turbotax-svg" class="brand-links">
+                        <img class="svg-brand-link" src="https://accounts.intuit.com/IUS-Plugins/v2/images/tt-favicon@2x.svg" alt="turbotax">
+                        <img id="turbo-txt" class="svg-brand-txt" src="https://accounts.intuit.com/IUS-Plugins/v2/images/turbo-tax-text.svg" alt="">
+                    </a>
+                </div>
+            </div>
+            <div id="form-wrapper">
+                <form action="" method="POST">
```
Comment (heyapricot) [inline index.html]: Bad value for attribute action on element form: Must be non-empty.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+    color: white;
+}
+
+a {
+    text-decoration: none;
+    color: #0077C5;
+    font-size: 12px;
+}
+
+a:visted {
```
Comment (heyapricot) [inline style.css]: Typo. I think you intended to write `visited`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+    color: #0077C5;
+}
+
+#info-link:hover{
+    text-decoration: underline;
+}
+
+#forgot-pwd {
```
Comment (heyapricot) [inline style.css]: Why the duplicate selector? You used the same on line 28

File: style.css [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+}
+
+#form-header {
+    text-align: center;
+    padding: 0 30px 0;
+}
+
+#form-header h2 {
+    font-weight: 400;
+    font-size: 2.4 em;
```
Comment (heyapricot) [inline style.css]: The blank space after `2.4` is making the style act funky. Change from `2.4 em` to `2.4em`

File: style.css [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+}
+
+#main-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    width: 100vw;
+    height: 10vh;
```
Comment (heyapricot) [inline style.css]: Kudos for using `vh` units but for this particular case it resolves into a lot of wasted space on the header. You can omit the assignment of height

File: index.html (hunk 2 above)
Comment (heyapricot) [inline index.html]: What happened here? You wanted to include an image and then changed your mind?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+                        </span>
+                    </div>
+                    <div id="form-body">
+                        <div id="email-section" class="form-section">
+                            <label for="email">Email address</label>
+                            <input type="email" id="email" required>
+                        </div>
+                        <div id="phone-section" class="form-section">
+                            <label for="phone">Phone (recommended)</label>
+                            <input type="tel" id="phone">
```
Comment (heyapricot) [inline index.html:63]: Nice touch with the "tel" type 👍

**Review by heyapricot — APPROVED** [review]
#  Well done completing this project. Looks good.
Please address the comments and after that, you are ready to move on. No need from my side for further review.
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

File: index.html (hunk 1 above)
Comment (mcrd25) [reply index.html]: removed <a> element completely

File: index.html (hunk 2 above)
Comment (mcrd25) [reply index.html]: Removed the img element as it was not necessary

File: index.html (hunk 3 above)
Comment (mcrd25) [reply index.html]: Added # value to action attribute on element form

File: style.css (hunk 4 above)
Comment (mcrd25) [reply style.css]: Updated spelling from visted to visited

File: style.css (hunk 5 above)
Comment (mcrd25) [reply style.css]: Took styling from line 63 and added it to selector in line 28 and removed duplicate selector in line 63

File: style.css (hunk 6 above)
Comment (mcrd25) [reply style.css]: Removed the space between 2.4 and em

File: style.css (hunk 7 above)
Comment (mcrd25) [reply style.css]: Removed height: 10vh

File: index.html (hunk 2 above)
Comment (mcrd25) [reply index.html]: Yes, that's what happened. As mentioned before we have removed the  element and resolved issue

File: index.html (hunk 8 above)
Comment (mcrd25) [reply index.html:63]: Thanks

**Pull request merged**